buildScripts/rename-artifact.js

```diff
@@ -21,8 +21,8 @@ fs.readFile(configFile, 'utf8', (err, xml) => {
 
     const currentVersion = obj.widget['$'].version;
     const artifactDir = 'build';
-    const defaultArtifactName = 'DrivingExaminerService.ipa';
-    const newArtifactName = `DrivingExaminerService-${currentVersion}-${Math.floor(Date.now() / 1000)}-${git.short()}.ipa`;
+    const defaultArtifactName = 'DES 3.ipa';
+    const newArtifactName = `DES3-${currentVersion}-${Math.floor(Date.now() / 1000)}-${git.short()}.ipa`;
```

Comment:
why is it 'DES 3' everywhere else and 'DES3' here.  Can we keep it consistant and have 'DES 3' so it matches 'DES 4' as well